Watching for file changes with StatReloader
Internal Server Error: /
Traceback (most recent call last):
  File "/Users/thibh/template-partials/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/thibh/template-partials/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/thibh/template-partials/demo/views.py", line 7, in index
    return render(request, 'demo/index.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/thibh/template-partials/.venv/lib/python3.11/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/thibh/template-partials/.venv/lib/python3.11/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/thibh/template-partials/.venv/lib/python3.11/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/thibh/template-partials/.venv/lib/python3.11/site-packages/django/template/backends/django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/thibh/template-partials/.venv/lib/python3.11/site-packages/django/template/engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/thibh/template-partials/.venv/lib/python3.11/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/thibh/template-partials/.venv/lib/python3.11/site-packages/template_partials/loader.py", line 41, in get_template
    template = self.loaders[0].get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/thibh/template-partials/.venv/lib/python3.11/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/thibh/template-partials/.venv/lib/python3.11/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "/Users/thibh/template-partials/.venv/lib/python3.11/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/thibh/template-partials/.venv/lib/python3.11/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/Users/thibh/template-partials/.venv/lib/python3.11/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/Users/thibh/template-partials/.venv/lib/python3.11/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/thibh/template-partials/.venv/lib/python3.11/site-packages/template_partials/templatetags/partials.py", line 109, in partialdef_func
    return _define_partial(parser, token, "endpartialdef")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/thibh/template-partials/.venv/lib/python3.11/site-packages/template_partials/templatetags/partials.py", line 128, in _define_partial
    raise template.TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: 'partialdef' tag requires 2-3 arguments
[07/Aug/2025 15:47:33] "GET / HTTP/1.1" 500 150261
